Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/redhat-pipeline-service/settings-operator:latest
```

Comment:
Most developers using the controller or making changes to it would rather build and deploy the image to their own repo (eg: quay.io/bnr/settings-operator). And the ones who just want to try it out or use the defaults can use the default image too.
And about the CI point too, I think even with this in place, the CI can still pass the URL with a specific tag which would override the default value.